src/server/db/migrations/schema_0.0.1.md

```diff
@@ -0,0 +1,117 @@
+# Data Schema version: `0.0.1`
```

Comment:
As we talked just now, this data schema documentation fits better into the top "/docs" directory. Please create a README in this migration directory to describe:

1. the goal of this migration;
2. what is changed. Some of your beautiful code comment in the migration script could go in this section;
2. Migration plan (how to perform the migration) including migration steps, what to prepare for each step; what output to expect for each step.